content/webapp/pages/search/index.tsx

```diff
@@ -515,28 +634,31 @@ const NewSearchPage: NextPageWithLayout<NewProps> = ({
                         </NextLink>
                       </CatalogueResultsSection>
                     </>
-                  ) : null}
+                  )}
                 </CatalogueResultsInner>
               </CatalogueResults>
             )}
-            <ContentResults data-testid="search-content-results">
-              {contentResults?.results?.map(result => (
-                <Space
-                  key={`${result.id}${result.highlightTourType || ''}`}
-                  $v={{ size: 'xl', properties: ['margin-bottom'] }}
-                >
-                  <ContentSearchResult {...result} />
-                </Space>
-              ))}
 
-              {contentResults?.totalPages ? (
-                <Pagination
-                  totalPages={contentResults.totalPages}
-                  ariaLabel="Content search results pagination"
-                  isHiddenMobile={false}
-                />
-              ) : null}
-            </ContentResults>
+            {contentResults && (
+              <ContentResults data-testid="search-content-results">
+                {contentResults?.results?.map(result => (
+                  <Space
+                    key={`${result.id}${result.highlightTourType || ''}`}
+                    $v={{ size: 'xl', properties: ['margin-bottom'] }}
+                  >
+                    <ContentSearchResult {...result} />
+                  </Space>
+                ))}
+
+                {contentResults?.totalPages && (
```

Comment:
```suggestion
                {!!contentResults?.totalPages && (
```

I was seeing 0 printed to the page, so coercing value into a boolean

Reply:
[addressed in latest commit](https://github.com/wellcomecollection/wellcomecollection.org/pull/11652/commits/bf334465572b4f00dd8975ac512225706ac5b614) and did it across other conditionals as well!